modules/nlu/src/backend/typings.ts

```diff
@@ -194,3 +194,10 @@ type SlotDefinition = Readonly<{
   name: string
   entities: string[]
 }>
+
+export class TrainingCanceledError extends Error {
+  constructor() {
+    super('Training cancelled')
+    this.name = 'CancelError'
+  }
+}
```

Comment:
add new line at the end of file (prettier should do it automatically, please chek that you have it installed in your vscode extensions)